configs/stash-cache/config.d/40-stash-cache-plugin.cfg

```diff
@@ -65,7 +65,7 @@ fi
 if defined ?~XC_RAMSIZE
    set eRAMSIZE=$XC_RAMSIZE
 else
-   set eRAMSIZE=1g
+   set eRAMSIZE=16g
```

Comment:
What is this ram used for?

Reply:
For the write queue. So it acts as a buffer between the origin and the disk caches. If the cache disks are too busy (i.e fully used the write queue) then the blocks are not being written to disk.

Reply:
Why would I want 16GB of buffered writes?  I remember a discussion a few years ago about this value, and we decided that a small size is better.